### PR TITLE
docs: corrected typo in nuxt v3 documentation

### PR DESCRIPTION
**Pull request by MoustaphaDev**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Corrects a typo in the Nuxt 3 docs where "Nuxt 2" is referenced instead of "Nuxt 3"

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/bridge/1.overview.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ Nuxt 3 natively supports TypeScript and ECMAScript Modules. Please check [Native
 - Remove `@nuxt/content` (1.x). A rewrite for Nuxt 3 is planned (2.x)
 - Remove `nuxt-vite`: Bridge enables same functionality
 - Remove `@nuxt/typescript-build`: Bridge enables same functionality
-- Remove `@nuxt/typescript-runtime` and `nuxt-ts`: Nuxt 2 has built-in runtime support
+- Remove `@nuxt/typescript-runtime` and `nuxt-ts`: Nuxt 3 has built-in runtime support
```
Comment (pi0) [inline docs/content/bridge/1.overview.md:162]: This is intentionally correct. Migration guide is about Bridge and Nuxt 2

Comment (MoustaphaDev) [reply docs/content/bridge/1.overview.md:162]: Well I thought that it's *Nuxt 3* that inititated built-in ts runtime support, isn't it ?

Comment (pi0) [reply docs/content/bridge/1.overview.md:162]: Runtime support is for for nuxt.config, server middleware and modules. We started to support runtime ts when migrated to jiti (can't recall what version of Nuxt 2 exactly but long ago!)

Comment (pi0) [reply docs/content/bridge/1.overview.md:162]: Nuxt3 has built-in build-time support for ts. Removing dependency on @nuxt/typescript-build

Comment (MoustaphaDev) [reply docs/content/bridge/1.overview.md:162]: Understood! Guess I'm not familiar enough with Nuxt. I'm gonna do more research next time before submitting a PR.

Comment (pi0) [reply docs/content/bridge/1.overview.md:162]: No worries. Looking forward for next contributions @MoustaphaDev <3

**Pull request closed without merging**